lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/client/YarnJobSubmissionHandler.java

```diff
@@ -90,19 +93,23 @@ public void close() {
   @Override
   public void onNext(final JobSubmissionEvent jobSubmissionEvent) {
 
-    LOG.log(Level.FINEST, "Submitting job with ID [{0}]", jobSubmissionEvent.getIdentifier());
+    LOG.log(Level.FINER, "Submitting {0} job with ID {1}",
+        new Object[] {this.isDriverUnmanaged ? "UNMANAGED" : "regular", jobSubmissionEvent.getIdentifier()});
 
     try (final YarnSubmissionHelper submissionHelper =
              new YarnSubmissionHelper(this.yarnConfiguration, this.fileNames, this.classpath, this.tokenProvider)) {
 
-      LOG.log(Level.FINE, "Assembling submission JAR for the Driver.");
-      final Optional<String> userBoundJobSubmissionDirectory =
-          getUserBoundJobSubmissionDirectory(jobSubmissionEvent.getConfiguration());
+      LOG.log(Level.FINEST, "Assembling submission JAR for the Driver.");
```

Comment:
This change looks too big on GitHub. Shouldn't we just see `FINE` replaced with `FINEST`?